src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I'd say use the least events possible with React and Redux.. The app will react to a change in the store if needed anyway.. Thoughts @alavers?